cms/envs/devstack.py

```diff
@@ -7,6 +7,9 @@
 from os.path import abspath, dirname, join
 
 from .production import *  # pylint: disable=wildcard-import, unused-wildcard-import
+from openedx.core.lib.features_setting_proxy import FeaturesProxy
```

Comment:
```suggestion
from openedx.core.lib.features_setting_proxy import FeaturesProxy

from .production import *  # pylint: disable=wildcard-import, unused-wildcard-import
```
to make pylint happy